_posts/2020-09-22-data-explorer-kafka-connect.md

```diff
@@ -0,0 +1,413 @@
+---
+layout: post
+title:  "Data Ingestion into Azure Data Explorer using Kafka Connect"
+date: 2020-09-22
+author: abhirockzz
```

Comment:
can you add yourself to the `authors.yml` file please?

Reply:
done! thanks for pointing that out